helpers/browser-helper.js

```diff
@@ -0,0 +1,33 @@
+import {fail, check} from "k6";
+import {browser} from 'k6/experimental/browser';
+
+export class BrowserHelper {
+    constructor(urlHelper) {
+        this.urlHelper = urlHelper;
+    }
+
+    async getLoggedInUserContext() {
+        const loginPage = browser.newPage();
+        await loginPage.goto(`${this.urlHelper.getStorefrontBaseUrl()}/en/login`);
+
+        await Promise.all([
+            loginPage.locator('#loginForm_email').type('sonia@spryker.com'),
```

Comment:
* I would suggest storing such data in the environment variables.

Reply:
I've introduced functionality for getting email and password from env in my task. How about releasing it as it if this tasks will be released first and I fix it in my task OR if my task will be released first we reuse it in this one?